src/views/HomeView.jsx

```diff
@@ -1,19 +1,38 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import { Link } from "react-router-dom";
 
-const HomeView = ({ match }) => {
+const HomeView = ({ match, uname }) => {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
```

Comment:
Can you move this into a custom hook in the /state folder?

Something like `use-users-list.js`, that works like the `use-backend` (that we should rename to "use-user-data") or similar...

Reply:
I understand. I'll work on that now.